haas/server.py

```diff
@@ -43,14 +45,13 @@ def stop_orphan_consoles():
     These may exist if HaaS was shut down uncleanly.
     """
     # Stop all orphan console logging processes on startup
-    db = model.Session()
-    nodes = db.query(model.Node).all()
+    nodes = db.session.query(model.Node).all()
```

Comment:
Would the pattern of `model.Node.query.all()` be a little more readable (here and other places), similar to flask/sqlalchemy's [quickstart](http://flask-sqlalchemy.pocoo.org/2.1/quickstart/)?